server/src/logic.js

```diff
@@ -419,14 +416,17 @@ export const getHandlers = ({ models, creators: Creators }) => {
         return org.getUsers();
       },
       tags(org, args) {
-        if (args.filter) {
-          return org.getTags({ where: { name: { [Op.like]: `%${args.filter}%` } }, order: [['id', 'ASC']] });
+        if (args.nameFilter && !args.typeFilter) {
```

Comment:
I think we can do better than defining the queries twice... I'll write some pseudo-code, see if you can make it work:
```
const where = {};
if (args.nameFilter) {
  where.name = { [Op.like]: `%${args.nameFilter}%` };
}
if (args.typeFilter) {
  where.type = { [Op.like]: `%${args.typeFilter}%` };
}
return org.getTags({ where, order: [['id', 'ASC']] };
```
^ this has the benefit of defining the parts of the where clause only once, assuming it will gladly take an empty where (pretty sure it will). It also means that an empty filter results in a sorted list of tags (your current implementation does not).

Reply:
it started as pseudo code and became real code, forgot to update comment.